front/src/applications/operationalStudies/components/Scenario/ScenarioDescription.tsx

```diff
@@ -33,7 +62,22 @@ const ScenarioDescription = ({
 
       <div className="scenario-description">
         {scenario.description && (
-          <div className="scenario-details-description">{scenario.description}</div>
+          <div
+            ref={descriptionRef}
+            title={isOpenedDescription ? '' : scenario.description}
+            className={`scenario-details-description ${isOpenedDescription ? 'opened' : 'not-opened'}`}
+            style={{ width: isOpenedDescription ? width + 120 : width }}
```

Comment:
Why do we need JS to synchronize the width here? Can't we use the same CSS rule as the sidebar? Something like

```css
width: calc(33% + 120px);
max-width: 100%;
```

Reply:
You're right, it's done